### PR TITLE
Add generator cloudwatch logs

### PR DESCRIPTION
**Pull request by inokappa**
Hi, @k1LoW 

Otsukaresama desu.

I added Generator CloudWatch Logs.
And fixed when filter is not specified.

Please confirm. :bow:

Regards.

Yohei

### CODE REVIEW

**Comment by inokappa** [comment]
oh...

memo: https://github.com/lostisland/faraday/pull/680

**Comment by inokappa** [comment]
oh... 😢 

memo: https://github.com/lostisland/faraday/issues/679

File: lib/awspec/generator/spec/cloudwatch_event.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ class CloudwatchEvent
       include Awspec::Helper::Finder
       def generate_all
         events = select_all_cloudwatch_events
-        raise 'Not Found alarm' if events.empty?
+        raise 'Not Found event' if events.empty?
```
Comment (k1LoW) [inline lib/awspec/generator/spec/cloudwatch_event.rb]: 👍

**Comment by k1LoW** [comment]
Hi @inokappa ! Thank you for your commit.

LGTM! 👍

**Comment by k1LoW** [comment]
Released as v0.73.2.

**Pull request merged**